index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Concerts</title>
+    <link rel="stylesheet" href="assets/css/base.css">
+    <link rel="stylesheet" href="assets/css/homepage.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"/>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous"/>
+  </head>
+  <body id="body">
+    <div id="top-bar">
+      <ul>
+        <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+        <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">LogOut</a></li>
+      </ul>
+    </div>
+    <header>
+      <div class="nav-logo">
+        <a href="#">Mega<br>Concerts</a>
+      </div>
+      <nav>
+        <div class="nav-links">
+          <ul class="nav-links-list">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li id="campain"><a href="#" id="campain-link">CC Campaign</a></li>
+          </ul>
```

Comment:
When the user is on the about page on the mobile version, he cannot return to the home page from mobile navigation.

- [ ] Kindly, implement a link in the mobile navigation so that the user can return to the home page from the about page on the mobile version. :100: